enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DependencyEnricher.java

```diff
@@ -182,6 +178,32 @@ private void removeTemplateObjects(List<HasMetadata> list, List<HasMetadata> obj
 
     }
 
+    public void filterAndAddItemsToBuilder(KubernetesListBuilder builder, List<HasMetadata> items) {
+        HashMap<KindAndName, Integer> aIndexMap = new HashMap<>();
```

Comment:
Insted of `Integer` use `HasMetadata` as value type
Map<KindAndName, HasMetadata>